pandas/tests/extension/base/reduce.py

```diff
@@ -66,6 +67,15 @@ def test_reduce_series(self, data, all_numeric_reductions, skipna):
             warnings.simplefilter("ignore", RuntimeWarning)
             self.check_reduce(s, op_name, skipna)
 
+    @pytest.mark.parametrize("skipna", [True, False])
+    def test_reduce_and_wrap(self, data, all_numeric_reductions, skipna):
```

Comment:
```suggestion
    def test_reduce_frame(self, data, all_numeric_reductions, skipna):
```

(to name it for what it is meant for user-facing, matching the naming of the other series test?)

Reply:
Yeah, I like that better, I've also renamed `check_reduce_and_wrap` to `check_reduce_frame`.